samples/frameworks/electron/ts/README.md

```diff
@@ -0,0 +1,80 @@
+
+### Register a new application in AAD and assign the "Azure Service Bus Data Owner" and "Azure Storage Blob Data Contributor" role to it.
+
+Authentication will still need to be set-up manually using the following instructions:
+
+- See https://docs.microsoft.com/azure/active-directory/develop/quickstart-register-app
+  to register a new application in the Azure Active Directory.
+- Note down the client id and tenant id from the above step.
+  You will need to set these in the constants.ts file later.
+
+For Electron we also need to configure platform settings correctly, and we will use `msal://` as the custom scheme. To set that up in your app registration:
+
+1. Click on Authentication in the app registration side bar
+2. Select "Add a platform"
+3. Select "Mobile and Desktop Application"
+4. In the "Custom redirect URIs" field please enter: msal://redirect
+5. Click "Configure"
+
+In your app registration, you will also need to add a permission for the `Microsoft.ServiceBus` and `Azure Storage` apps.
+When adding permission for `Microsoft.ServiceBus` and `Azure Storage`, the type should be `delegated permissions` and the permission should be `user_impersonation`.
+
+Finally, as mentioned in the header you'll need to make sure your application has the "Azure Service Bus Data Owner" and the "Azure Storage Blob Data Contributor" roles assigned to it. Please refer to the [Service Bus Role Assignment][servicebusaad] and [Azure Storage Role Assignment][storageaad] for up-to-date instructions on how to configure those.
+
+## Running the sample
+
+Once the above steps are completed you'll want to ensure Electron has the necessary constants defined. To do this, open `src/constants.ts` and provide the necessary values to configre the application. Only empty strings need to be replaced with actual values.
+
+Install the various packages as well as the TypeScript compiler using:
+
+```bash
+npm install
+```
+
+Run the sample app:
+
+```bash
+npm start
+```
+
+A new electron window will open containing a single Log In button. Once you login you'll be able to send and receive Service Bus messages as well as fetch and display a text file from Azure Blob Storage.
+
+## Next Steps
+
+Take a look at our [API Documentation][apiref] for more information about the APIs that are avaiable.
+
+[electron]: https://www.electronjs.org/
+[typescript]: https://www.typescriptlang.org/docs/home.html
+[freesub]: https://azure.microsoft.com/free
+[servicebus]: https://docs.microsoft.com/javascript/api/@azure/service-bus
+[storageblob]: https://docs.microsoft.com/javascript/api/@azure/storage-blob
+[identity]: https://docs.microsoft.com/javascript/api/@azure/identity
+[apiref]: https://docs.microsoft.com/javascript/api/
+[servicebusaad]: https://docs.microsoft.com/en-us/azure/service-bus-messaging/authenticate-application
```

Comment:
These two links have broken CI link check. Please see following for future reference.I will submit a PR to fix them now.
Check here for more information: https://aka.ms/azsdk/guideline/links

DO NOT include locale /en-us/ information in links: https://docs.microsoft.com/en-us/azure/service-bus-messaging/authenticate-application.

Reply:
Thanks for letting me know, if you'd prefer I can submit a PR to fix these so you don't have to - just let me know.